apps/communications/contacts/test/marionette/list_test.js

```diff
@@ -51,6 +51,68 @@ marionette('Contacts > List', function() {
     });
   });
 
+  suite('List Order > ', function() {
+    var data = [['GG', 'E'], ['AA', 'Z'],
+      ['XX', 'C'], ['CC', 'X'], ['EE', 'G'], ['FF', 'F'], ['HH', 'D'],
+      ['BB', 'Y'], ['YY', 'B'], ['ZZ', 'A'], ['DD', 'H']];
+    setup(function() {
+      data.forEach(contactData => {
+        subject.addContact({
```

Comment:
We already have a way of performing the injection, will write the test using it, instead of doing the whole flow.